docker/development-easy-macos/README.md

```diff
@@ -0,0 +1,14 @@
+### Easy Development Docker Environment for macOS with M1, M2, M* SoCs
+The instructions for The Easy Development Docker environment can be found at [CONTRIBUTING.md](../../CONTRIBUTING.md#code-contributions-local-development).
+
+#### File Permissions
+
+UIDs and GIDs are handled differently between macOS and Linux. This discrepancy will cause permission errors on .git/objects/pack directory
+ files similar to:
+
+```
+development-easy-macos-openemr-1     | chown: /var/www/localhost/htdocs/openemr/.git/objects/pack/pack-ab1e32999500a2e52117c503b8126b3f77e3ff26.pack: Permission denied
+```
+
+To remedy this, change the file permissions on those files to be more permissive. For example, you can execute `chmod -R 777 ./git/objects/pack`; however, we must note that this level of permissiveness is discouraged in production environments as it goes against the principle of least privilege best practice. 
```

Comment:
I have tried to run this command while in the openemr/docker/development-easy directory. And receive the following error response "chmod: ./git/objects/pack: No such file or directory"

Reply:
@toduyemi sorry, the command I put in the README is incorrect. It should be `chmod -R 777 .git/objects/pack`. I accidentally put in a ./, a relative path to the non-existent `git` folder instead of the hidden `.git` folder.

Let me know if that works for you. My colleague and I are using OpenEMR in a class we're teaching this semester, and this worked for our students using macOS.